cassdegrees/ui/views.py

```diff
@@ -158,6 +161,49 @@ def sampleform(request):
         return render(request, 'sampleform.html')
 
 
+def create_program(request):
+    # If POST request, redirect the received information to the backend:
+    render_properties = {
+        'msg': None,
+        'is_error': False
+    }
+
+    if request.method == 'POST':
+        post_data = request.POST
+
+        degree_dict = \
+            {
+                'code': post_data.get('code'),
+                'name': post_data.get('name'),
+                # Do some early validation of these fields
+                'year': int(post_data.get('year')),
+                'units': int(post_data.get('units')),
+                'degreeType': post_data.get('degreeType')
+            }
+
```

Comment:
```suggestion

        for k, v in degree_dict.items():
            render_properties[k] = v
            
```

Same changes as Josh, allows inputted values to appear in the form once it has been re-submitted